static/src/stylesheets/module/identity/_manage-account-tab.scss

```diff
@@ -269,6 +269,30 @@
             }
         }
     }
+
+    /* Cancellation Contribution section
+    ===================================================================== */
+    .contribution-cancellation__menu {
+        border-top: solid 1px black;
+
+        h3 {
+            margin-top: 28px;
+            margin-bottom: 26px;
+        }
+
+        contribution-cancellation__menu__actions {
+            margin-top: 56px;
```

Comment:
It would be great to use multiples of `$gs-baseline` for these vertical margin values